server/routes/offenceCodeDecisions/offenceCodeDecisionsValidation.ts

```diff
@@ -0,0 +1,21 @@
+import { FormError } from '../../@types/template'
+
+type DecisionForm = {
+  selectedDecisionId?: string
+}
+
+type ErrorType = 'MISSING_SELECTION'
+
+const errors: { [key in ErrorType]: FormError } = {
+  MISSING_SELECTION: {
+    href: '#selectedDecisionId',
+    text: 'Please make a choice', // TODO correct message
```

Comment:
Not sure what this message should be at the moment.

Reply:
LGTM